src/components/forms/formComponent.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../../layouts/forms/form.css'
+const FormBox = (props)=>{
+    return(
+        <div className='wl-st-form-box'>
+            <label className='info-name'>{props.name}</label><br></br>
+            <br></br>
+            <input type={props.type} placeholder={props.ph} className='info-val'></input><br></br>
```

Comment:
Isn't there input field directly given my mui?

Reply:
yes sir,there is a textfield component. I'll update it